app/views/students/_form.html.erb

```diff
@@ -0,0 +1,393 @@
+      </div>
+    </div>
+
+    <div class="row">
+      <div class="col-md-12">
+        <p class="help-block"> At least one phone number is required.</p>
+      </div>
+
+      <div class="col-md-4">
+        <%= f.input :home_phone, maxlength: 14, placeholder: "(###) ###-####", required: false, input_html: { class: "mask_phone" } %>
+      </div>
+
+      <div class="col-md-4">
+        <%= f.input :cell_phone, maxlength: 14, placeholder: "(###) ###-####", required: false, input_html: { class: "mask_phone" } %>
+      </div>
+
+      <div class="col-md-4">
+        <%= f.input :other_phone, maxlength: 14, placeholder: "(###) ###-####", required: false, input_html: { class: "mask_phone" } %>
+      </div>
+    </div>
+
+    <div class="row">
+      <div class="col-md-6">
+        <%= f.input :email, as: :string, label: "Email", maxlength: 255, required: false, input_html: { class: "mask_email" } %>
+      </div>
+    </div>
+
+    <div class="row">
+      <div class="col-md-4">
+        <%= f.input :emergency_contact_name, maxlength: 255, required: false %>
+      </div>
+
+      <div class="col-md-4">
+        <%= f.input :emergency_contact_phone, maxlength: 14, placeholder: "(###) ###-####", required: false, input_html: { class: "mask_phone" } %>
+      </div>
+    </div>
+    
+  <section id="personalInfo">
+    <h4>Personal Information</h4>
+    <hr>
+    <div class="row">
+      <div class="col-md-4">
+        <div class="form-group">
+          <label class="control-label radio-label"> Gender </label>
+          <%= f.collection_radio_buttons :gender, [[:female, 'Female'] ,[:male, 'Male'], [:other, 'Other']], :first, :last %>
+        </div>
+      </div>
+
+      <div class="col-md-2">
+        <div class="form-group">
+          <label class="control-label radio-label"> Hispanic or Latino </label>
+          <%= f.collection_radio_buttons :hispanic_or_latino, [[:true, 'Yes'] ,[:false, 'No']], :first, :last %>
+        </div>
+      </div>
+
+      <div class="col-md-3">
+        <div class="form-group">
+          <%= f.input :date_of_birth, input_html: { class: 'datepicker' }, placeholder: "YYYY-MM-DD", required: false %>
+        </div>
+      </div>
+
+      <div class="col-md-3">
+        <%= f.input :status, collection: status_options, required: false %>
+      </div>
+    </div>
+
+    <div class="row">
+      <div class="col-md-4">
+        <%= f.input :native_language, collection: native_language, selected: "English", required: false %>
+      </div>
+
+      <div class="col-md-4">
+        <%= f.input :race, collection: race, required: false %>
+      </div>
+
+      <div class="col-md-4">
+        <%= f.input :country_of_birth, as: :string, maxlength: 255, required: false %>
+      </div>
+    </div>
+  </section>
+
+    <hr>
+
+    <div class="row">
+      <div class="col-md-12">
+        <div class="form-group">
+          <label class="control-label"> Contact Method Preferences </label>
```

Comment:
I think Contact Method Preferences is a sub-section of the Personal Information preferences, hence the different style.  The style is the same in the tutor from